src/schema/resolvers/signup_resolver.js

```diff
@@ -0,0 +1,43 @@
+
+import models from '../../models'
+import createToken from '../../auth'
+
+const resolvers = {
+  Mutation: {
+    signup (parent, { data }, context) {
+      const { username, password, lastName, firstName } = data
+      if (!username || !password) return { error: 'No password or username' }
+      const user = models.user.findOrCreate({
+        where: { username: username },
+        defaults: { password: password, lastName: lastName, firstName: firstName }
```

Comment:
You can write this like this: `defaults: { password, lastName, firstName }`. When the key has the same name than the value you can do that.